### PR TITLE
Draft for HIPPA compliance

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

Adds information about Appwrite's HIPPA compliance and steps taken to ensure safety of PHI.
## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/docs/advanced/security/+page.markdoc [hunk 1]
```diff
@@ -29,9 +29,8 @@ Coming soon
 {% /cards_item %}
 
 {% cards_item href="/docs/advanced/security/hippa" title="HIPPA" %}
-Coming soon.
+Appwrite is HIPPA compliant. Learn about how Appwrite's measures to protect PHI.
```
Comment (eldadfux) [inline src/routes/docs/advanced/security/+page.markdoc:32]: What is PHI can we first list the full term?

File: src/routes/docs/advanced/security/hippa/+page.markdoc [hunk 2]
```diff
@@ -1,12 +1,28 @@
 ---
 layout: article
-title: HIPPA (Coming soon)
+title: HIPPA
```
Comment (eldadfux) [inline src/routes/docs/advanced/security/hippa/+page.markdoc:3]:
```suggestion
title: HIPAA
```

and in all other places

Comment (eldadfux) [reply src/routes/docs/advanced/security/hippa/+page.markdoc:3]: Also the icon file name, to help avoid future confusion.

**Review by eldadfux — CHANGES_REQUESTED** [review]
Is there way for to include the phrase `Is Appwrite HIPAA compliant?` for SEO?

File: src/routes/docs/advanced/security/hippa/+page.markdoc (hunk 2 above)
Comment (gewenyu99) [reply src/routes/docs/advanced/security/hippa/+page.markdoc:3]: Updated

File: src/routes/docs/advanced/security/+page.markdoc (hunk 1 above)
Comment (gewenyu99) [reply src/routes/docs/advanced/security/+page.markdoc]: Done, list as full term.

It's also used in the HIPAA docs, but we introduce the term.

File: src/routes/docs/advanced/security/hippa/+page.markdoc (hunk 2 above)
Comment (gewenyu99) [reply src/routes/docs/advanced/security/hippa/+page.markdoc]: And updated elsewhere.

**Review by eldadfux — APPROVED** [review]

**Pull request merged**